### DISCUSSION TITLE
disabling bleak logging

### DISCUSSION THREAD

**Discussion by net-far** [hbldh/bleak#1213] [Q&A] [answered]
* bleak version: 22.3.1
* Python version: 3.7.9
* Operating System: windows 11
* BlueZ version (`bluetoothctl -v`) in case of Linux:

### Description

I am using bleak to connect to my device, in my code i use the logging module for python.
I get "Services resolved for BleakClientWinRT" on my logging file as intended but i cant seem to disable bleak logging to no longer get bleak internal logger  

### What I Did

I found the following:
https://bleak.readthedocs.io/en/latest/troubleshooting.html#enable-logging
I tried both in PowerShell and cmd to do as instructed but still got the "Services resolved for BleakClientWinRT" on my logging file
after that i tried to change the client.py  here:  logger.info("Services resolved for %s", str(self)) but obviously it did not work 

i need help disabling the logging in bleak

**Comment by dlech** [comment, marked as answer]
You can use `bleak_logger = logging.getLogger("bleak")` to get a Logger object for Bleak and then `bleak_logger.setLevel(logging.WARNING)` to change the logging level. 

Also happy to take a PR to change this particular instance from `logger.info` to `logger.debug` since this seems like it should just be a debug message.

**Reply by net-far** [reply]
it seems that in the latest develop branch the line( logger.info("Services resolved for %s", str(self)) as well as self._services_resolved is no longer in use
